system-x/services/hashicorp-vault/src/main/java/software/tnb/hashicorp/vault/resource/local/LocalHashicorpVault.java

```diff
@@ -0,0 +1,63 @@
+package software.tnb.hashicorp.vault.resource.local;
+
+import software.tnb.common.deployment.Deployable;
+import software.tnb.common.deployment.WithDockerImage;
+import software.tnb.hashicorp.vault.service.HashicorpVault;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.auto.service.AutoService;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@AutoService(HashicorpVault.class)
+public class LocalHashicorpVault extends HashicorpVault implements Deployable, WithDockerImage {
```

Comment:
`WithDockerImage` already defined in `HashicorpVault`, so it doesn't need to be here (but depending on the ocp implementation)